### PR TITLE
Fix locale switcher prettyurls

### PR DESCRIPTION
**Pull request by JOGUI22**
- Fixes minor problems in map with the new navbar.
- Fix Language switcher on blog posts
- Jordi casadevall member info

### CODE REVIEW

**Review by sergiogc9 — APPROVED** [review]

File: lib/utils.ts [hunk 1]
```diff
@@ -10,7 +10,21 @@ export function capitalizeFirstLetter(text: string): string {
...
+export const decodeMdxFilePathData = (
+  filePath: string,
+  lang: string
+): { id: string; urlPath: string } => {
+  const cleanPath = removeFilePathExtension(filePath);
+  const id = cleanPath.split('-')[0];
+
+  return {
+    id: id,
+    urlPath: `/${lang}/blog/${cleanPath}`,
```
Comment (llorenspujol) [inline lib/utils.ts:26]: I would suggest removing the id from the urlPath. I think this way the url is cleaner, and there is no real need, as far as I know, to have the id in the final path.

File: config/member.ts [hunk 2]
```diff
@@ -36,9 +36,11 @@ const members: Member[] = [
   {
     github: 'JOGUI22',
     name: 'Jordi Casadevall',
-    role: 'Platform Engineer',
+    role: 'Senior Software Engineer',
```
Comment (llorenspujol) [inline config/member.ts:39]: Platform was cool...😆

File: app/[lang]/map/map.css [hunk 3]
```diff
@@ -116,7 +116,7 @@ p {
 }
 
 .map__card.map__card--fullscreen {
-  max-height: 90%;
+  max-height: 85%;
```
Comment (llorenspujol) [inline app/[lang]/map/map.css:119]: nice 👌

**Review by llorenspujol — CHANGES_REQUESTED** [review]
Nice PR! 👏 

I would suggest only removing the id from the blog urls! 
Let me know your thoughts about it

File: lib/utils.ts (hunk 1 above)
Comment (sergiogc9) [reply lib/utils.ts:26]: I think he's using the id to identify the blog here:
https://github.com/gironajs/gironajs.com/pull/8/files#diff-cf7e29e4902dea0aa000da86f34225ba6d7a6d13c982343f255b15f827269199R23

Comment (JOGUI22) [reply lib/utils.ts]: Id deleted: https://github.com/gironajs/gironajs.com/pull/8/commits/b618f5b0f651f3349d93a8a343953df695475cd4

**Review by llorenspujol — APPROVED** [review]

**Pull request merged**